### ISSUE TITLE
Takes forever to start plex

### ISSUE THREAD

**Issue by fletchto99** [cjmurph/PmsService#10]
Running plex normally with windows it will start up in about 30 second max, and the web interface is available. When I use it as a service it takes > 5 minutes just for the web interface to become available. Is this due to a miss configuration in my plex server settings?

**Comment by cjmurph** [comment]
Hi,

Is it running under the same user account when run as a service vs normally? where is the media located? where is the plex library located? The service simply starts plex just like you would normally, but it does run in session 0 with different resources available (no mapped drive letters for example).

**Comment by fletchto99** [comment]
@cjmurph The media is located in the machine I'm running (3 different HDD's). Resources shouldn't be an issue as this is one of the few external services I have starting up during boot. Plex it's self is running off of an SSD so it should load relativity quick. Running outside of the service wrapper it loads pretty much instantly when I log in.

The problem is I want plex to run with out me having to log in, this way I can start up my machine remotely and be able to use plex.

A user on the Plex forum suggested I grab the logs generated by the service. I've also tried scheduled tasks and NSSM all yielding the same result, really slow initial load times.

For reference I also have a thread on the Plex forums: http://forums.plex.tv/discussion/207685/plex-takes-forever-to-start-as-a-scheduled-task - Note that this is referring to the task scheduler method I tried, but they all do the same thing.

**Comment by cjmurph** [comment]
So this is all with the same user account? Have you moved the plex library files out of the users folder or have you left them in the default location? Personally I think I'd be looking at the plex logs not the service logs.

Try this:
1. Open Task Manager.
2. Using the service tray application, stop plex
3. Confirm in task manager that PlexMediaServer.exe isn't running.
4. Using the service tray application, start plex and look for the PlexMediaServer.exe to appear in task manager. If you notice it come up straight away, but you don't have web access yet, go through the plex media server logs and see whats going on.

It might be worth grabbing a copy of the logs after a fast local start so you have something to compare to.

**Comment by fletchto99** [comment]
Yeah they are all on the same user @cjmurph. I will attempt doing what you just recommended! Will report back shortly

**Comment by fletchto99** [comment]
Found the issue. After double checking the logs (like you recommended) and doing a few tests I noticed that when running plex by myself it would open up iTunes but also be instantly available. When running as a service it wouldn't open iTunes (which makes sense). So I thought perhaps when it's running as a service it's failing to open itunes and getting stuck on that. After some googling I came across: http://superuser.com/questions/948157/itunes-starts-automatically-at-startup-on-windows-10
And tada that fixes the issue. PMS, or Scheudled tasks or NSSM all work instantly now as if I was running it myself.
